traefik/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - traefik
 
+## 1.1.0 / 2023-06-13
```

Comment:
Technically there was a 1.0.0 release (it happened [here](https://github.com/DataDog/integrations-extras/pull/663)) but that release was just a re-release of 0.2.0 but upgrading the number to 1.0.0 instead.